qiskit/compiler/assemble.py

```diff
@@ -242,10 +242,12 @@ def _parse_pulse_args(backend, qubit_lo_freq, meas_lo_freq, qubit_lo_range,
         except (ModelValidationError, AttributeError):
             from collections import namedtuple
             backend_config_defaults = getattr(backend_config, 'defaults', {})
-            BackendDefault = namedtuple('BackendDefault', ('qubit_freq_est', 'meas_freq_est'))
+            BackendDefault = namedtuple('BackendDefault',
+                                        ('qubit_freq_est', 'meas_freq_est', 'rep_time'))
```

Comment:
These lines actually don't have to be changed (lines 245-250)

Reply:
To clarify, the `try` statement is a relic from an earlier time. It should be removed in a separate PR.

Reply:
I'll open a PR now... :)